test/runtests.jl

```diff
@@ -20,7 +20,29 @@ cd(dirname(@__FILE__)) do
 
     @everywhere include("testdefs.jl")
 
-    reduce(propagate_errors, nothing, pmap(runtests, tests; err_retry=false, err_stop=true))
+    const max_worker_rss = parse(Int, get(ENV, "JULIA_TEST_MAXRSS_MB", "200"))
```

Comment:
Okay, I see. That is much easier to reason about directly from the new test output.